src/ledger/LedgerManagerImpl.cpp

```diff
@@ -504,6 +504,19 @@ LedgerManagerImpl::syncMetrics()
     mApp.syncOwnMetrics();
 }
 
+void
+LedgerManagerImpl::emitAnyStoredSafeMetaAfterLedgerValidation()
```

Comment:
I'm a little iffy on this name -- maybe `emitDelayedMetaFromLastLedger`?

Reply:
Sure, I'd be happy with that.  (I'm certainly not attached to `emitAnyStoredSafeMetaAfterLedgerValidation`!)

Reply:
(This method is private now, and can now be called to emit either delayed or immediate metadata, so I've just given it the name `emitNextMeta`.)